code/crates/core-consensus/src/handle/decide.rs

```diff
@@ -18,6 +18,12 @@ where
     // We only decide proposals for the current height
     assert_eq!(height, state.driver.height());
 
+    // Cancel the commit timeout
+    perform!(
+        co,
+        Effect::CancelTimeout(Timeout::commit(consensus_round), Default::default())
+    );
+
```

Comment:
imo it is a bit more clear to cancel in `decide_current_no_timeout()` when we do value sync. We can be here via normal consensus with commit timer expiration so no need to cancel.

Reply:
Ok, then we can close this I guess.